### PR TITLE
When extracting code files, check if file is writable before try to copy

### PR DESCRIPTION
**Pull request by voldemortensen**
and don't report success if any file isn't copied to current directory

### CODE REVIEW

File: php/commands/core.php [hunk 1]
```diff
@@ -226,9 +228,18 @@ private static function _copy_overwrite_files( $source, $dest ) {
 					mkdir( $dest_path );
 				}
 			} else {
-				copy( $item, $dest . DIRECTORY_SEPARATOR . $iterator->getSubPathName() );
+				if ( ! $item->isWritable() ) {
```
Comment (danielbachhuber) [inline php/commands/core.php]: Shouldn't this be `if ( $item->isWritable() )` ?

**Comment by voldemortensen** [comment]
Removing the erroneous not check completely broke my code... I'll refactor and push a new fix.

**Comment by voldemortensen** [comment]
Sorry for the first 4 commits. Should be good now though.

**Comment by voldemortensen** [comment]
Now that this is working properly, what else needs to happen to get this merged?

**Comment by danielbachhuber** [comment]
As much as I'd like to have tests for this, it would involve adding an unwritable file to the test directory, which will probably break things.

**Pull request merged**